test/quantization/fx/test_numeric_suite_fx.py

```diff
@@ -681,9 +680,9 @@ def _op_is_unmatchable(op):
                 self.assertTrue(
                     _op_in_base_sets_of_related_ops(base_op),
                     f"{base_op} not in sets of related ops")
-            else:
+            elif not _op_in_base_sets_of_related_ops(base_op):
```

Comment:
@vkuzo this is the change I made to the test, please take a look

Reply:
basically we don't explicitly check the quantize handler type in this branch, but just directly check if op is in the related ops set